test/test_issue837.py

```diff
@@ -0,0 +1,433 @@
+# Recovered from https://github.com/RDFLib/rdflib/pull/1068
+from rdflib import ConjunctiveGraph, Dataset, Graph, URIRef
+
+michel = URIRef("urn:michel")
+tarek = URIRef("urn:tarek")
+bob = URIRef("urn:bob")
+likes = URIRef("urn:likes")
+hates = URIRef("urn:hates")
+pizza = URIRef("urn:pizza")
+cheese = URIRef("urn:cheese")
+
+timblcardn3 = """
```

Comment:
May be worth adding this in a separate file if we want consistent test data to use in multiple tests.

Reply:
Agreed. I have a few consistent test data sources in the id-as-ctx work which can usefully sit in there, so going with the obvious I created ta subdirectory `consistent_test_data` and added timbl's card to that.